.github/renovate.json

```
@@ -20,6 +20,23 @@
 		{
 			"paths": [ "packages/codesniffer/composer.json" ],
 			"rangeStrategy": "replace"
+		},
+		{
+			"groupName": "Instant Search Dependency Updates",
+			"packageNames": [
```

Looks like they recently renamed "packageNames" to "matchPackageNames" (renovatebot/renovate#6939)
```suggestion
			"matchPackageNames": [
```

For that matter, we might want to update lines 17 and 21 similarly.